### ISSUE TITLE
Cell Click Action: allow specify none and several columns

### ISSUE THREAD

**Issue by devaskim** [thingsboard/thingsboard#11906]
**Is your feature request related to a problem? Please describe.**
I need to trigger the same custom action after clicking on on all columns or on subset of columns.

**Describe the solution you'd like**
1) let specify several columns in  `Column index` configuration option
2) let do not set  `Column index` configuration option that will mean triggering this action for all visible columns.

**Describe alternatives you've considered**
Copy-paste cell click action, !!!BUT!!! not having `Duplicate Action` function (like it did for dashboard state duplication) it is boring story. Here is feature request https://github.com/thingsboard/thingsboard/issues/11907

_____________________________________________________
**Disclaimer** 

We appreciate your contribution whether it is a bug report, feature request, or pull request with improvement (hopefully). Please comply with the [Community ethics policy](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies)), and do not expect us to answer your requests immediately. Also, do not treat *GitHub issues* as a support channel.

**Comment by ssakharova** [comment]
Hello @devaskim,

If you need to trigger the same widget action after clicking on all cells in all columns, you can just use the "Action cell button" option.

As for the "Duplicate action button" feature, this is indeed a valid improvement that I've already added to our backlog. There's no set timeframe for our dev reviews; however, please follow our [Release Notes](https://thingsboard.io/docs/reference/releases/) to see all new additions to the platform in the subsequent releases.

**Comment by devaskim** [comment]
> If you need to trigger the same widget action after clicking on all cells in all columns, you can just use the "Action cell button" option.

@ssakharova No, I can't, it is another scenario. Despite action implementation is the same, it depends on input parameters which are different for each clicked cell.

**Comment by ssakharova** [comment]
@devaskim Okay, I decided to create an internal prod ticket for your suggestion to improve the "On cell click" widget action. Our developers need to review it and decide if this improvement is necessary and if it can be implemented in future versions of ThingsBoard.
Please follow our [Release Notes](https://thingsboard.io/docs/reference/releases/) for updates on new additions to the platform in future releases.
